data/sql/updates/pending_db_world/rev_1729179186645856952.sql

```diff
@@ -0,0 +1,5 @@
+-- Looting the 3 keys is annoying as a group, since the stones appear ready to loot but
+-- only allow looting after a while after the previous pickup
+-- Objects: Burning Key, Cresting Key & Thundering Key from Stones of Binding
+
+UPDATE `gameobject_template` SET `Data2` = 0 WHERE `entry` IN (2689,2690,2691);
```

Comment:
This fix is incorrect, the `Data2` values for these objects on latest AC already match a 3.4.0 47168 sniff.

Reply:
Hm, annoying, since the behavior with a long wait for a 2nd looter seems illogical.

So my fix here can be scrapped, but I wonder if I'm the only one with this issue, or if it works as intended?